pkgs/applications/misc/neo/default.nix

```
@@ -0,0 +1,26 @@
+{ lib, stdenv, fetchurl, ncurses }:
+
+stdenv.mkDerivation rec {
+  pname = "neo";
+  version = "0.6";
+
+  src = fetchurl {
+    url =
+      "https://github.com/st3w/${pname}/releases/download/v${version}/${pname}-${version}.tar.gz";
```

```suggestion
    url = "https://github.com/st3w/neo/releases/download/v${version}/neo-${version}.tar.gz";
```
Url on one line is more consistent with formatting in nixpkgs. 

Substituting the name with `pname` only makes it more unreadable imo and it's longer than the 3 letters `neo` has.

---

Committed.